pkg/monitortests/clusterversionoperator/terminationmessagepolicy/monitortest.go

```diff
@@ -79,7 +113,36 @@ func (w *terminationMessagePolicyChecker) CollectData(ctx context.Context, stora
 
 	// existingViolations is the list of violations already present, don't add to it once we start enforcing
 	existingViolations := map[string]sets.String{
-		"namespace": sets.NewString("pods/<name> <containerType>[<containerName>]"),
+		"namespace": sets.NewString("<containerType>[<containerName>]"),
+		"openshift-catalogd": sets.NewString(
+			"pods/catalogd-controller-manager",
+		),
+		"openshift-cluster-api": sets.NewString(
+			"pods/capa-controller-manager",
+			"pods/capi-controller-manager",
+		),
+		"openshift-cluster-csi-drivers": sets.NewString(
+			"containers[openstack-cinder-csi-driver-operator]",
+			"containers[azure-file-csi-driver-operator]",
+			"containers[shared-resource-csi-driver-operator]",
+			"containers[vmware-vsphere-csi-driver-operator]",
+			"containers[vsphere-webhook]",
+			"pods/kubevirt-csi-node",
+			"pods/nutanix-csi-controller",
+			"pods/nutanix-csi-node",
+			"pods/ibm-vpc-block-csi-controller",
+			"pods/ibm-vpc-block-csi-node",
+			"pods/powervs-block-csi-driver-operator",
+		),
```

Comment:
tidy list produced from https://search.dptools.openshift.org/?search=must+have+terminationMessagePolicy%3D&maxAge=48h&context=1&type=junit&name=4.16.*&excludeName=upgrade&maxMatches=5&maxBytes=20971520&groupBy=job

@jsafrane

Reply:
I think this covers all of them except for kubevirt and nutanix:
https://github.com/openshift/cluster-storage-operator/pull/470
https://github.com/openshift/vmware-vsphere-csi-driver-operator/pull/231
https://github.com/openshift/ibm-vpc-block-csi-driver-operator/pull/114